.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
delete this please, better use
git reset --soft HEAD^
git push origin IA-04_Antoniuk_Anton -f